### PR TITLE
fix console route

### PR DESCRIPTION
**Pull request by ajssmith**
I tested it with and without cert-manager and appeared to work correctly but I may be missing something...

### CODE REVIEW

File: pkg/resources/routes/route.go [hunk 1]
```diff
@@ -41,9 +41,9 @@ func NewRouteForCR(m *v1alpha1.Interconnect, listener v1alpha1.Listener) *routev
 	if listener.SslProfile != "" {
 		route.Spec.TLS = &routev1.TLSConfig{
 			Termination:                   routev1.TLSTerminationPassthrough,
-			InsecureEdgeTerminationPolicy: routev1.InsecureEdgeTerminationPolicyNone,
+			InsecureEdgeTerminationPolicy: routev1.InsecureEdgeTerminationPolicyRedirect,
 		}
-	} else {
+	} else if !listener.Http {
```
Comment (grs) [inline pkg/resources/routes/route.go]: I'm confused... edge termination I think only makes sense for the http listener. I think this is also where the change in InsecureEdgeTerminationPolicy from Allow to Redirect makes sense (for passthrough non-tls http won't succeed).

Comment (ajssmith) [reply pkg/resources/routes/route.go]: Nope, I'm the one confused :)

If the listener does not have an sslProfile (e.g. the http listener) why would the route spec have a TLS block?

Comment (grs) [reply pkg/resources/routes/route.go]: We can set edge termination, which means traffic outside the cluster is still TLS.

Comment (ajssmith) [reply pkg/resources/routes/route.go]: Ahh, lightbulb went off...as you mentioned earlier, port then changes to 443. I think i understand now.

Comment (fgiorgetti) [reply pkg/resources/routes/route.go]: And listener has to be "http", right?

**Pull request merged**